File: docs/components_page/components/layout/vertical.jl
using DashBootstrapComponents

row = html_div(
    [
        dbc_row(
            [
                dbc_col(html_div("One of three columns")),
                dbc_col(html_div("One of three columns")),
                dbc_col(html_div("One of three columns")),
            ],
            align = "start",
        ),
        dbc_row(
            [
                dbc_col(html_div("One of three columns")),
                dbc_col(html_div("One of three columns")),
                dbc_col(html_div("One of three columns")),
            ],
            align = "center",
        ),
        dbc_row(
            [
                dbc_col(html_div("One of three columns")),
                dbc_col(html_div("One of three columns")),
                dbc_col(html_div("One of three columns")),
            ],
            align = "end",
        ),
        dbc_row([
            dbc_col(html_div("One of three columns"), align = "start"),
            dbc_col(html_div("One of three columns"), align = "center"),
            dbc_col(html_div("One of three columns"), align = "end"),
        ]),
    ],
    className = "pad-row",
);

File: docs/components_page/components/layout/vertical.py
import dash_bootstrap_components as dbc
from dash import html

row = html.Div(
    [
        dbc.Row(
            [
                dbc.Col(html.Div("One of three columns")),
                dbc.Col(html.Div("One of three columns")),
                dbc.Col(html.Div("One of three columns")),
            ],
            align="start",
        ),
        dbc.Row(
            [
                dbc.Col(html.Div("One of three columns")),
                dbc.Col(html.Div("One of three columns")),
                dbc.Col(html.Div("One of three columns")),
            ],
            align="center",
        ),
        dbc.Row(
            [
                dbc.Col(html.Div("One of three columns")),
                dbc.Col(html.Div("One of three columns")),
                dbc.Col(html.Div("One of three columns")),
            ],
            align="end",
        ),
        dbc.Row(
            [
                dbc.Col(html.Div("One of three columns"), align="start"),
                dbc.Col(html.Div("One of three columns"), align="center"),
                dbc.Col(html.Div("One of three columns"), align="end"),
            ]
        ),
    ],
    className="pad-row",
)
